### PR TITLE
Views REST API

### PR DESCRIPTION
**Pull request by johan-bjareholt**
Do not merge before https://github.com/ActivityWatch/aw-core/pull/12

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
The tests fail because it tries to import query and views from aw_core which is not merged yet.

Works fine locally on my machine.

**Comment by ErikBjare** [comment]
Re-running the tests now that https://github.com/ActivityWatch/aw-core/pull/12 is merged.

File: aw_server/api.py [hunk 1]
```diff
@@ -202,12 +227,84 @@ def post(self, bucket_id):
...
+"""
+
+    VIEWS
+
+"""
+
+
+@api.route("/0/views")
+class ViewListResource(Resource):
+    """
```
Comment (ErikBjare) [inline aw_server/api.py]: What's up with these empty docstrings?

File: aw_server/api.py [hunk 2]
```diff
@@ -27,15 +27,21 @@
...
 fDuration = api.model('Duration', {
     'value': fields.Float(),
     'unit': fields.String(),
 })
 
-# TODO: Move to aw_core.models, construct from JSONSchema (if reasonably straight-forward)
 event = api.model('Event', {
     'timestamp': fields.List(fields.DateTime(required=True)),
-    'duration': fields.List(fields.Nested(fDuration)),
+    # Duration validation is broken
```
Comment (johan-bjareholt) [inline aw_server/api.py]: Do not remember why i made that comment, reverted and it seems to work fine.

File: aw_server/api.py [hunk 3]
```diff
@@ -107,13 +125,20 @@ def post(self, bucket_id):
...
+
+
 @api.route("/0/buckets/<string:bucket_id>/events")
 class EventResource(Resource):
     """
     Used to get and create events in a particular bucket.
     """
 
-    @api.marshal_list_with(event)
+    #@api.marshal_list_with(event)
```
Comment (johan-bjareholt) [inline aw_server/api.py]: Do not remember why i made that comment, reverted and it seems to work fine.

File: aw_server/api.py (hunk 1 above)
Comment (johan-bjareholt) [inline aw_server/api.py]: Fixed!

**Comment by ErikBjare** [comment]
Looks good to me! Merging.

**Pull request merged**